tests/sentry/replays/test_organization_replay_index.py

```diff
@@ -832,41 +832,27 @@ def test_archived_records_are_not_returned(self):
         with self.feature(REPLAYS_FEATURES):
             response = self.client.get(self.url)
             assert response.status_code == 200
-            assert len(response.json()["data"]) == 0
-
-    def test_archived_records_not_returned_with_environment_filtered(self):
```

Comment:
these two tests don't seem to test any behavior, on master they pass even if archived is set to false